### DISCUSSION TITLE
GitLive uses reth as a testbed due to its's rapid pace of development

### DISCUSSION THREAD

**Discussion by nbransby** [paradigmxyz/reth#12839] [General]
Hi, I'm the founder of GitLive and we have been using your repo for testing because it's one of the fastest moving projects we've come across on GitHub - so congrats for that! 🚀🚀  

We are working on enhancing our pre-emptive merge conflict detection algorithm and had a question about your workflow @ reth. In particular I wanted to ask @emhane or others if they are using branch stacking (branch off an existing branch to continue working whilst the previous branch is awaiting review).  

It's not obvious by simply looking at the git history so thought I would reach out, any details on your branching strategies would help us improve GitLive and would be greatly appreciated! 

![image](https://github.com/user-attachments/assets/3f2eea86-fe92-4668-9df9-f34b88228c9a)

**Comment by onbjerg** [comment]
Yes, some of us use stacked branches, but it's not a requirement:) Generally our workflow is branching off from main, doing a unit of work, pull requesting that in. Some choose to add additional work on top of open PRs, making them stacked, but it's not always the case.

We exclusively squash merge into main

**Reply by nbransby** [reply]
Thanks for the details, the squashing explains some of the behavior we're seeing that needs some tweaking.  I would love for you guys to try GitLive and let us know your feedback if you think it could benefit your team.
